sdf/supershape.hlsl

```diff
@@ -0,0 +1,21 @@
+#include ../space/cart2polar.hlsl
+#include ../math/superFormula.hlsl
+
+/*
+author: Kathy McGuiness
+description: Returns 2D supershape
+use: supershapeSDF(<vec2> st, <float> size s, <float> a, <float> b, <float> n1, <float> n2, <float> n3, <float> m)
```

Comment:
Would be possible to have some description of what each one of the arguments is for `a`, `b`, `m` and `n<N>` seams a bit abstract.

Reply:
Are supershape/superFormula well known terms? I guess it depends on how many people have watched Daniel Shiffman's Supershape coding challenge. https://thecodingtrain.com/challenges/23-2d-supershapes.

The beautiful thing about the supershape is that by adjusting parameters, you can obtain an almost infinite variety of shapes.  
I totally agree about the parameters and can try my best.  The resource I am going from is Paul Bourke's website.
http://paulbourke.net/geometry/supershape/

The 3D supershape (which is also very cool) also uses the superformula.  I separated it out because it was stated that  library was granular.

Reply:
perfect! that's good enough, if two functions will use it it's a solid reason to keep it modular.  Love the reference to http://paulbourke.net/geometry/supershape/ Thanks you for that. Also love the reference to @shiffman ! Providing this links to the description will be great for reference. Again, thank you!